rhino/src/main/java/org/mozilla/javascript/Context.java

```diff
@@ -1414,7 +1426,8 @@ final Script compileString(
             ErrorReporter compilationErrorReporter,
             String sourceName,
             int lineno,
-            Object securityDomain) {
+            Object securityDomain,
+            Consumer<CompilerEnvirons> compilerEnvironsProcessor) {
```

Comment:
Same here -- this is a public method in a public class that is definitely used outside this project and we need to keep it for compatibility.

Reply:
This is _not_ public, which is why we have not added the "backward compatibility" method.